### PR TITLE
Updates to Validator Docs

### PR DESCRIPTION
**Pull request by nooomski**
As per #1154

### CODE REVIEW

File: docs/validators/overview.md [hunk 1]
```diff
@@ -8,31 +8,31 @@ order: 1
...
 
 ## Set Up a Website
 
-Set up a dedicated validator's website and signal your intention to become a validator on the Cosmos Forum [Validator Candidates](https://forum.cosmos.network/t/validator-candidates-websites/127/3) site. Posting your validator website is important because delegators want to have information about the entity they are delegating their ATOM to.
+We suggest you set up a dedicated validator's website and signal your intention to become a validator in the [Cosmos Discord](https://discord.gg/cosmosnetwork). Posting your validator website can help because delegators want to have information about the entity they are delegating their ATOM to.
 
 ## Seek Legal Advice
 
-Seek legal advice if you intend to run a Validator.
+We recommend you seek legal advice if you intend to run a Validator.
```
Comment (yaruwangway) [inline docs/validators/overview.md]: run a Validator  -> run a validator node

File: docs/validators/overview.md [hunk 2]
```diff
@@ -8,31 +8,31 @@ order: 1
...
 
-Validators are expected to equip their datacenter location with redundant power, connectivity, and storage backups. Expect to have several redundant networking boxes for fiber, firewall, and switching and then small servers with redundant hard drive and failover. Hardware can be on the low end of datacenter gear to start out with.
+Validators are expected to equip their datacenter location with redundant power, connectivity, and storage backups. Expect to have several redundant networking boxes for fiber, firewall, and switching and then small servers with redundant hard drive and failover.
 
-Initial network requirements can be low. The current testnet requires minimal resources. Then bandwidth, CPU, and memory requirements rise as the network grows. Large hard drives are recommended for storing years of blockchain history.
+You can find the minimum hardware requirements on the [instructions for joining mainnet](../hub-tutorials/join-mainnet.md). As the network grows, bandwidth, CPU, and memory requirements rise. Large hard drives are recommended for storing years of blockchain history.
```
Comment (yaruwangway) [inline docs/validators/overview.md]: Large hard drives -> Large hard drives and large capacity of RAM

File: docs/validators/validator-faq.md [hunk 3]
```diff
@@ -16,18 +16,20 @@ The [Cosmos Hub](../getting-started/what-is-gaia.md) is based on [Tendermint](ht
 
 ### What is staking?
 
-The Cosmos Hub is a public Proof-Of-Stake (PoS) blockchain, meaning that the weight of validators is determined by the amount of staking tokens (ATOM) bonded as collateral. These ATOM can be self-delegated directly by the validator or delegated to the validator by other ATOM holders.
+The Cosmos Hub is a public Proof-Of-Stake (PoS) blockchain, meaning that the weight of validators is determined by the amount of staking tokens (ATOM) bonded as collateral. These ATOM tokens can be self-delegated directly by the validator or delegated to the validator by other ATOM holders.
```
Comment (yaruwangway) [inline docs/validators/validator-faq.md:19]: the weight of validators is -> the weights of validators are

File: docs/validators/validator-faq.md [hunk 4]
```diff
@@ -16,18 +16,20 @@ The [Cosmos Hub](../getting-started/what-is-gaia.md) is based on [Tendermint](ht
...
+The weight (i.e. voting power) of a validator determines whether they are an active validator. The active validator set is limited to [an amount](https://www.mintscan.io/cosmos/validators) that changes over time.
 
 ### What is a full node?
 
-A full node is a program that fully validates transactions and blocks of a blockchain. A full node is distinct from a light node that processes only block headers and a small subset of transactions. Running a full node requires more resources than a light node. A full node is required in order to be a validator. In practice, running a full node implies running a non-compromised and up-to-date version of the software with low network latency and with no downtime.
+A full node is a program that fully validates transactions and blocks of a blockchain. A full node is distinct from a pruned node that processes only block headers and a small subset of transactions. Running a full node requires more resources than a pruned node. A full node is required in order to be a validator. In practice, running a full node implies running a non-compromised and up-to-date version of the software with low network latency and with no downtime.
```
Comment (yaruwangway) [inline docs/validators/validator-faq.md]: validator can also prune.

Comment (yaruwangway) [reply docs/validators/validator-faq.md]: validator even with pruning will have to retain a certain amount of blocks, even validator sets `min-retention-blocks` to 1, validator will still have to retain a certain amount of blocks which is enough for validating present block.

File: docs/validators/validator-faq.md [hunk 5]
```diff
@@ -81,38 +85,39 @@ The Tendermint key and the application key are derived from account keys that ar
 
 After a validator is created with a `create-validator` transaction, the validator is in one of three states:
 
-- `in validator set`: Validator is in the active set and participates in consensus. Validator is earning rewards and can be slashed for misbehavior.
+- `in validator set`: Validator is in the active set and participates in consensus. The validator is earning rewards and can be slashed for misbehavior.
 - `jailed`: Validator misbehaved and is in jail, i.e. outside of the validator set. 
 
-  - If the jailing is due to being offline for too long, the validator can send an `unjail` transaction in order to re-enter the validator set. 
+  - If the jailing is due to being offline for too long (i.e. having missed more than `95%` out of `10,000` blocks), the validator can send an `unjail` transaction in order to re-enter the validator set.
```
Comment (yaruwangway) [inline docs/validators/validator-faq.md]: `95%` out of `10,000` blocks -> `95%` out of the last`10,000` blocks (so the max allowed not online blocks is not accumulated from different `10,000` blocks right ?)

File: docs/validators/validator-setup.md [hunk 6]
```diff
@@ -167,26 +130,21 @@ gaiad query tendermint-validator-set | grep "$(gaiad tendermint show-address)"
...
+
 ## Common Problems
 
 ### Problem #1: My validator has `voting_power: 0`
 
-Your validator has become jailed. Validators get jailed, i.e. get removed from the active validator set, if they do not vote on `500` of the last `10000` blocks, or if they double sign. 
+Your validator has become jailed. Validators get jailed, i.e. get removed from the active validator set, if they do not vote on at least `500` of the last `10000` blocks, or if they double sign. 
 
-If you got jailed for downtime, you can get your voting power back to your validator. First, if `gaiad` is not running, start it up again:
+If you got jailed for downtime, you can get your voting power back to your validator. First, if you're not using [Cosmovisor](https://docs.cosmos.network/master/run-node/cosmovisor.html) and `gaiad` is not running, start it up again:
```
Comment (yaruwangway) [inline docs/validators/validator-setup.md:147]: validator also need to unjail themselves `gaiad tx slashing unjail --from`

File: docs/validators/validator-faq.md (hunk 5 above)
Comment (nooomski) [reply docs/validators/validator-faq.md]: Correct! Added

File: docs/validators/validator-setup.md (hunk 6 above)
Comment (nooomski) [reply docs/validators/validator-setup.md:147]: The link to #unjail validator specifies the command :)

File: docs/validators/validator-faq.md (hunk 4 above)
Comment (nooomski) [reply docs/validators/validator-faq.md]: Thanks! Specified it in the text.

File: docs/validators/validator-setup.md [hunk 7]
```diff
@@ -2,10 +2,10 @@
 order: 2
 -->
 
-# Run a Validator on the Cosmos Hub Mainnet
+# Running a Validator
 
 ::: tip
-Information on how to join the mainnet (`genesis.json` file and seeds) is held [in our `launch` repo](https://github.com/cosmos/launch/). 
+We suggest you try out joining a public testnet first. Information on how to join the Vega testnet can be found [here]](https://github.com/cosmos/vega-test/blob/master/public-testnet/README.md).
```
Comment (glnro) [inline docs/validators/validator-setup.md]:
```suggestion
We suggest you try out joining a public testnet first. Information on how to join the Vega testnet can be found [here](https://github.com/cosmos/vega-test/blob/master/public-testnet/README.md). 
```

When we merge the updated joining testnet docs, I might also recommend adding that here or replacing the link

File: docs/validators/validator-setup.md [hunk 8]
```diff
@@ -2,10 +2,10 @@
...
 
-# Run a Validator on the Cosmos Hub Mainnet
+# Running a Validator
 
 ::: tip
-Information on how to join the mainnet (`genesis.json` file and seeds) is held [in our `launch` repo](https://github.com/cosmos/launch/). 
+We suggest you try out joining a public testnet first. Information on how to join the Vega testnet can be found [here]](https://github.com/cosmos/vega-test/blob/master/public-testnet/README.md). 
 :::
 
 Before setting up your validator node, make sure you've already gone through the [Full Node Setup](../hub-tutorials/join-mainnet.md) guide.
```
Comment (glnro) [inline docs/validators/validator-setup.md]:
```suggestion
Before setting up a validator node, make sure to have completed the [Joining Mainnet](../hub-tutorials/join-mainnet.md) guide.
```

File: docs/validators/validator-setup.md (hunk 7 above)
Comment (nooomski) [reply docs/validators/validator-setup.md]: Good point, might as well just add it now as we should aim to have the new testnet up asap.

```suggestion
We suggest you try out joining a public testnet first. Information on how to join the most recent testnet can be found [here](../hub-tutorials/join-testnet.md). 
```

**Review by okwme — APPROVED** [review]
fantastic : )

**Pull request merged**